docs/source/author/chapters.rst

```diff
@@ -33,7 +31,7 @@ The ``content`` directory for the chapters application contains
 Content Files
 ==============================================================================
 
-There are 3 different types of files used for adding content to CS Unplugged:
+There are 3 different types of files used for adding content to the CS Field Guide:
 
 - Content Markdown files
```

Comment:
I think *somewhere* (probably right at the start of Author docs) we should say the field guide is intended to be developed in English and then translated to other languages